test/gscraper_web_web/controllers/page_controller_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule GscraperWebWeb.PageControllerTest do
+  use GscraperWebWeb.ConnCase
```

Comment:
Let's add `async: true`, same as `GscraperWebWeb.HomePage.ViewHomePageTest `

Reply:
I think the attribute `async: true` is used for View tests only, but this is the controller test 🤔
anw, this controller will be removed since it is auto-generated when initializing the project.